test/unit/cli-domain-get-components-deps.js

```diff
@@ -64,29 +70,32 @@ describe('cli : domain : get-components-deps', () => {
   });
 
   describe('custom templates and package dependencies with versions', () => {
-
     before(() => {
       const readJsonStub = sinon.stub();
 
       readJsonStub.onFirstCall().returns({
         oc: baseOcFragment('oc-template-react'),
-        dependencies: { 'oc-template-react': '1.2.3'}
+        devDependencies: { 'oc-template-react-compiler': '1.2.3' }
```

Comment:
can we add here `eslint` as devDependency too? So we can test that is not added to the ones that need to be installed